src/components/ebay-notice/index.marko

```diff
@@ -26,15 +28,17 @@ $ var isLightGuidance = isSection && !input.status;
         input.fillWindow && `${type}-notice--fill`,
         input.class
     ]>
-    <if(!isLightGuidance)>
+    <if(!isLightGuidance && (isIconVisible || isWindowTitle))>
         <${isPage ? input.a11yHeadingTag || "h2" : "span"}
             class=`${type}-notice__status`
             id:scoped="status">
-            <if(isWindow && input.title)>
-                <ebay-icon
-                    class="window-notice__icon"
-                    name="confirmation-filled"
-                    aria-label=input.a11yHeadingText/>
+            <if(isWindowTitle)>
```

Comment:
Is there an issue now with the icon showing up if windowTitle is false. I think it shows up on the `<else>`

Maybe we should figure out all the different states for this and try to fix it that way. I think it is still overly complex more than it should be. 

My initial guess for this is:
if lightguidance
   if title and window
       if image
          show image
        show title
     else if image
        show image
Something like that. But not sure all the different paths.

Reply:
Notice is going to get some serious refactoring in v6 to make this stuff easier to follow. Just do what you can for now until then.